conductor_api/src/conductor/base.rs

```diff
@@ -653,6 +605,77 @@ impl Conductor {
             })
     }
 
+    pub fn build_conductor_api(
+        &mut self,
+        instance_id: String,
+        config: &Configuration,
+    ) -> Result<IoHandler, HolochainError> {
+        let instance_config = config.instance_by_id(&instance_id)?;
+        let agent_id = instance_config.agent.clone();
+        let agent_config = config.agent_by_id(&agent_id)?;
+        let mut api_builder = ConductorApiBuilder::new();
+        // Signing callback:
+        if let Some(true) = agent_config.holo_remote_key {
+            // !!!!!!!!!!!!!!!!!!!!!!!
+            // Holo closed-alpha hack:
+            // !!!!!!!!!!!!!!!!!!!!!!!
+            api_builder = api_builder.with_outsource_signing_callback(
+                self.agent_config_to_id(&agent_config)?,
+                self.config
+                    .signing_service_uri
+                    .clone()
+                    .expect("holo_remote_key needs signing_service_uri set"),
```

Comment:
If we are returning a result, should we not handle this error better?

Reply:
This code was not changed but just moved. I chose an expect here because it is in the scope of the Holo closed-alpha hack and should go away soon. But maybe you're right and we should change it to return an error. I will go ahead and merge this anyway since this discussion is not really in scope for this PR.